crates/ethereum-forks/src/hardfork.rs

```diff
@@ -61,11 +61,17 @@ pub enum Hardfork {
     /// Canyon
     #[cfg(feature = "optimism")]
     Canyon,
+    /// ArbOS 11
+    #[cfg(feature = "arbitrum")]
+    ArbOS11,
     /// Cancun.
     Cancun,
     /// Ecotone
     #[cfg(feature = "optimism")]
     Ecotone,
+    /// ArbOS 20 Atlas
+    #[cfg(feature = "arbitrum")]
+    ArbOS20Atlas,
```

Comment:
I'd like to not add these and instead add these as a comment that these exists on arbitrum